.evergreen/scripts/run-auth-tests.sh

```diff
@@ -99,11 +99,21 @@ elif command -v otool >/dev/null; then
   LD_LIBRARY_PATH="${openssl_lib_prefix}" otool -L "${test_gssapi}" | grep "libssl" || true
 fi
 
+# TODO: Remove `skip_for_zseries` when resolving CDRIVER-5990.
+function skip_for_zseries() {
```

Comment:
```suggestion
skip_for_zseries() {
```
Redundant (+ [Bash Pitfall 25](https://mywiki.wooledge.org/BashPitfalls#function_foo.28.29); benign in this case but nevertheless).